varcode/variant.py

```diff
@@ -180,11 +180,11 @@ def reference_name(self):
         return self.ensembl.reference_name
 
     def __str__(self):
-        return "Variant(contig=%s, start=%d, ref=%s, alt=%s, reference_name=%s)" % (
+        return "Variant(contig='%s', start=%d, ref='%s', alt='%s', reference_name='%s')" % (
             self.contig,
             self.start,
-            self.ref if self.ref else ".",
-            self.alt if self.alt else ".",
+            self.ref,
```

Comment:
We check the length of both `alt` and `ref` in the initializer so I don't think it's possible for them to be `None` in a constructed `Variant`.